### PR TITLE
Migrate the Grafana changes to Theia

### PR DESCRIPTION
**Pull request by heanlan**
- Add indirect cycle handling to Grafana Sankey panel
- Limit max data values of Grafana panels

### CODE REVIEW

**Comment by heanlan** [comment]
I removed the configurable namespace from all dashboard.json. As in the documentation - [dashboard-customization](https://github.com/antrea-io/theia/blob/main/docs/network-flow-visibility.md#dashboards-customization), we ask the users to put the exported JSON file to the path `theia/build/charts/theia/provisioning/dashboards`. It's inconvenient and not straight-forward enough for them to replace all the Namespaces(e.g. flow-visibility) to '{{ .Release.Namespace }}'. 

cc @yanjunz97 for opinions

**Comment by yanjunz97** [comment]
> I removed the configurable namespace from all dashboard.json. As in the documentation - [dashboard-customization](https://github.com/antrea-io/theia/blob/main/docs/network-flow-visibility.md#dashboards-customization), we ask the users to put the exported JSON file to the path `theia/build/charts/theia/provisioning/dashboards`. It's inconvenient and not straight-forward enough for them to replace all the Namespaces(e.g. flow-visibility) to '{{ .Release.Namespace }}'.
> 
> cc @yanjunz97 for opinions

I see the concern. Do you think we need to add some words to remind users that if they install Grafana Flow Collector in a different namespace, they need to take care of the namespace changing in the json files?

**Comment by heanlan** [comment]
> I see the concern. Do you think we need to add some words to remind users that if they install Grafana Flow Collector in a different namespace, they need to take care of the namespace changing in the json files?

Thanks, added in the latest commit.

File: docs/network-flow-visibility.md [hunk 1]
```diff
@@ -554,14 +554,29 @@ visualization. They can be found in the Home page of Grafana, by clicking
...
+
+- Also note that we limit the number of values displayed on panels. For table
+panel on the Flow Records Dashboard, the limit is set to 10000. For Sankey
+diagrams, Chord diagrams, and time-series line graphs, the limit is 50. For
+pie charts, the limit is 25. The motivation is, when the input data is very
+large, we want to keep the charts readable and avoid consuming too much time
+and resources to render them.
+
+If you want to stop filtering traffic by Namespace, or edit the panel limit,
+you will need to edit the ClickHouse SQL query for each individual panel. Please
```
Comment (dreamtalen) [inline docs/network-flow-visibility.md:575]: `each` and `individual` may be redundant?

File: docs/network-flow-visibility.md [hunk 2]
```diff
@@ -554,14 +554,29 @@ visualization. They can be found in the Home page of Grafana, by clicking
...
+follow the [dashboards customization](#dashboards-customization) section for
+more information. As a special case, to edit the panel limit for pie charts,
+instead of editing the query, please follow the [doc](https://grafana.com/docs/grafana/latest/visualizations/pie-chart-panel/#limit)
```
Comment (dreamtalen) [inline docs/network-flow-visibility.md:578]: line wrap

**Review by dreamtalen — APPROVED** [review]
LGTM

**Pull request merged**